runtime/acala/src/lib.rs

```diff
@@ -1537,8 +1537,8 @@ pub fn create_x2_parachain_multilocation(index: u16) -> MultiLocation {
 parameter_types! {
 	pub const DOTCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::DOT);
 	pub const LDOTCurrencyId: CurrencyId = CurrencyId::Token(TokenSymbol::LDOT);
-	pub MinimumMintThreshold: Balance = 2 * dollar(DOT);
-	pub MinimumRedeemThreshold: Balance = 20 * dollar(LDOT);
+	pub MinimumMintThreshold: Balance = 50 * cent(DOT);
+	pub MinimumRedeemThreshold: Balance = 5 * dollar(LDOT);
```

Comment:
why are you reducing value here. it should be 10x the KSM value as KSM is ~10x the price of DOT